calc/src/App.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: #e5e5e5;
+}
+button {
```

Comment:
Normally it's good to create .button class instead of styling every button directly because in your app at the end you can have a lot of different buttons.